File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    boolean ans;
    public boolean isPalindrome(int x) {
        
        String str = Integer.toString(x);
        int n = str.length();
        char c1[] = str.toCharArray();
        for (int i=0; i<str.length();i++){
            if (c1[i]==c1[n-1-i]){ans= true;}
            else
            {ans=false;
            break;}
        }
      return ans;  
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        
        y = str(x)
        
        for i in range(len(y)/2):
            if y[i] != y[-1-i]:
                return False
            
        return True
        
                
            
        